Gui/opensim/build.xml

```diff
@@ -232,6 +232,8 @@
             value="${app.version}">
             <replacefilter token="@VERSION@" />
         </replace>
+        <chmod file="${mac.dist.inner.dir}/etc/OpenSim.conf" perm="744"/>
+        <chmod file="dist/OpenSim.app/Contents/Info.plist" perm="644"/>
```

Comment:
Can you add some comments (perhaps describing why you're using 744 for one and 644 for another) and a TODO indicating that this is temporary?